torch/testing/_internal/common_methods_invocations.py

```diff
@@ -13682,8 +13697,36 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
            supports_autograd=False,
            skips=(
            )),
+    OpInfo('randn',
+           dtypes=floating_and_complex_types_and(torch.half, torch.bfloat16, torch.complex32),
+           op=lambda *args, **kwargs: wrapper_set_seed(torch.randn, *args, **kwargs),
+           supports_out=True,
+           sample_inputs_func=sample_inputs_randn,
+           supports_autograd=False,
+           skips=(
+               # Tests that assume input is a tensor or sequence of tensors
+               DecorateInfo(unittest.skip("Test expects tensor input"), "TestCommon", "test_noncontiguous_samples"),
+               DecorateInfo(unittest.skip("Test expects tensor input"), "TestVmapOperatorsOpInfo", "test_vmap_exhaustive"),
+               DecorateInfo(unittest.skip("Test expects tensor input"), "TestVmapOperatorsOpInfo", "test_op_has_batch_rule"),
+               # Reference doesn't support the pin_memory parameter
+               DecorateInfo(unittest.expectedFailure, 'TestDecomp', 'test_comprehensive'),
```

Comment:
does `test_complrehensive` send `pinned=True` argument? that would be surprising.

Reply:
I agree, although I didn't investigate the details. The way the PR is current written setting `pin_memory=False` would also throw an error, which is overly restrictive

Reply:
fwiw, what we have done in other places is to accept the parameter and then write `assert not pinned_memory`.